meson.build

```diff
@@ -176,15 +176,14 @@ subdir('milter/client')
 subdir('milter/server')
 subdir('milter/manager')
 subdir('libmilter')
-libruby = dependency('ruby-3.2', 'ruby-3.1', 'ruby-3.0', 'ruby-2.7', 'ruby',
-                     required: true)
+libruby = dependency('ruby', required: true)
```

Comment:
Can we implement this feature something like the following?

```meson
libruby = dependency('ruby-3.2', required: false)
if not libruby.found()
  libruby = dependency('ruby-3.1', required: false)
endif
...
if not libruby.found()
   libruby = dependency('ruby', required: true)
endif
```

Reply:
I see!
How about this?: 225119e506b511b0cdd71a357f47539add3b8eeb

Reply:
You can also use `if meson.version().version_compare('>=0.60.0')` and use the newer API, which is simpler and has better logging, and only use a complex fallback on older versions of Meson.

Reply:
@daipom Could you follow the @eli-schwartz 's suggestion?
We can remove the code for old Meson easily when we require Meson 0.60.0 or later with the suggested approach.

Reply:
@eli-schwartz Thanks for the advice!
@kou Sure! I have fixed it!